docs/administration.emergency-users.md

```diff
@@ -50,44 +50,44 @@ When emergency access is no longer required the following steps will revert perf
 - Remove user from meshProject by opening the project again and navigating to **Project Access** > **Current Access** (performed automatically if expiration date was set)
 - Remove user from meshWorkspace via the [workspace control plane](./meshcloud.workspace.md#managing-your-meshworkspace): go to **Workspace Access** > **Current Access**.
 
-## Access with meshPartner
+## Access with Organization Admin
 
-Even when no workspace manager is available, users with partner admin/employee access can manage permissions for their managed meshWorkspaces.
+Even when no workspace manager is available, users with organization admin/user access can manage permissions for their managed meshWorkspaces.
 
-### Adding emergency user as meshPartner
+### Adding emergency user as Organization Admin
 
 Note: this only works when Workspace Manager role assignment is enabled via the panel. This depends on the `restrictCustomerAdminRoleAssignment` configuration
 value ([read more here](meshstack.onboarding.md#workspace-user-invitations))
 
-In order to manage users for a specific meshProject, the partner user must add their own account to the respective meshWorkspace as a workspace manager:
+In order to manage users for a specific meshProject, the admin user must add their own account to the respective meshWorkspace as a workspace manager:
 
-- Ensure that the meshPartner is selected from the meshWorkspace dropdown
+- Ensure that the meshAdmin is selected from the meshWorkspace dropdown
 - Open "Administration" from the settings menu in the top right
 - Navigate to "Workspaces" and select "Workspace User" from the actions column for the target workspace
 - Activate “Add Myself” button in the top right
 
 Afterwards they may proceed to manage users for this meshWorkspace as a workspace manager (see previous section).
 
-### Removing emergency user as meshPartner
+### Removing emergency user as Organization Admin
 
-Since the meshPartner user is now a workspace manager the procedure is the same as outlined in the previous section.
-However, to revert the temporary workspace manager role assignment, another workspace manager must remove the partner user from the workspace via **Workspace Access** > **Current Access**.
+Since the Admin user is now a workspace manager the procedure is the same as outlined in the previous section.
+However, to revert the temporary workspace manager role assignment, another workspace manager must remove the admin user from the workspace via **Workspace Access** > **Current Access**.
 
 ## Auditing Emergency Access
 
-Since meshWorkspace/meshProject access permissions should not be granted lightly all changes to them are logged and can be audited by partner admin/employee users.
+Since meshWorkspace/meshProject access permissions should not be granted lightly all changes to them are logged and can be audited by organization admin/user.
```

Comment:
```suggestion
Since workspace/project access permissions should not be granted lightly all changes to them are logged and can be audited by organization admin/user.
```